src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -21,7 +21,7 @@ internal static class PathUtilities
     {
         // We consider '/' a directory separator on Unix like systems. 
         // On Windows both / and \ are equally accepted.
-        internal static readonly char DirectorySeparatorChar = PlatformInformation.IsUnix ? '/' : '\\';
+        internal static readonly char DirectorySeparatorChar = Path.DirectorySeparatorChar;
```

Comment:
>internal static readonly char DirectorySeparatorChar = Path.DirectorySeparatorChar; [](http://example.com/codeflow?start=8&length=83)

Would it make sense to convert this to a property? It looks like we are storing value of one readonly field in another readonly field.